examples/sqs_queue_consumer.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+"""
+aiobotocore SQS Consumer Example
+"""
+
+# Boto should get credentials from ~/.aws/credentials or the environment
+import asyncio
+
+import aiobotocore
+
+QUEUE_NAME = 'test_queue1'
+
+
+@asyncio.coroutine
+def go(loop):
+    session = aiobotocore.get_session(loop=loop)
+    client = session.create_client('sqs', region_name='us-west-2')
+    response = yield from client.get_queue_url(QueueName=QUEUE_NAME)
+    assert response['ResponseMetadata']['HTTPStatusCode'] == 200
```

Comment:
normally if the call fails an exception is raised, can get_queue_url return a failed response w/o raising?  Like for s3.get_object, a 404 is raised as an exception.

Reply:
similar for the rest of the checks for 200

Reply:
Will check, I just spammed the assertions everywhere :wink: